src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
+            height: 100%;
+        }
+
+        .be {
+            height: 100%;
+
+            .bcs-activity-feed {
+                width: 100%;
+            }
+        }
+
+        .bcs-activity-feed-comment-input {
+            width: 100%;
+        }
+
+        .be.bcs {
+            flex-direction: column;
+            width: 100%;
+            min-width: unset;
+            max-width: unset;
+            height: 100%;
+            max-height: 48px;
+            -webkit-transition: max-height .5s ease-in-out 0s;
+            transition: max-height .5s ease-in-out 0s;
+        }
+
+        .bcs-content {
+            width: unset;
+            height: 100%;
+
+            .bcs-scroll-content {
+                width: unset;
+            }
+        }
+
+        .be.bcs.bcs-is-open {
+            position: absolute;
+            bottom: 0;
+            z-index: 2;
+            height: 100%;
+            max-height: 100%;
+            -webkit-transition: max-height .5s ease-in-out 0s;
+            transition: max-height .5s ease-in-out 0s;
+        }
+
+        .bcs {
+            .bcs-SidebarNav,
+            .bcs-SidebarNav-tabs {
+                flex-direction: row;
+            }
+        }
+
+        .bcs-SidebarNav-main {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .bcs-NavButton::before {
+            right: 0;
+            bottom: 0;
+            left: 0;
+            height: 2px;
+        }
+
+        .bcs .bcs-NavButton,
+        .bcs-SidebarNav-secondary .btn-plain,
+        .bdl-AdditionalTab {
+            width: 48px;
+            height: 48px;
+            margin-right: 12px;
+        }
+
+        .bdl-AdditionalTabs > *:not(.bdl-MoreOptions) {
+            display: none;
+        }
+
+        .bcs .bcs-SidebarNav-overflow .bdl-AdditionalTabs {
+            padding: 0;
+        }
+
+        .bcs .bcs-SidebarNav-overflow::after,
+        .bcs .bcs-SidebarNav-overflow::before {
+            display: none;
+        }
+
+        .bcs .bcs-SidebarNav-footer {
+            flex: 1 1 48px;
```

Comment:
What is the source of this value? Should it be a variable?

Reply:
Did some more testing and seems we don't need this property. I've removed it.